controllers/operator_controller_test.go

```diff
@@ -875,19 +875,19 @@ func verifyConditionsInvariants(op *operatorsv1alpha1.Operator) {
 
 var testEntitySource = input.NewCacheQuerier(map[deppy.Identifier]input.Entity{
 	"operatorhub/prometheus/0.37.0": *input.NewEntity("operatorhub/prometheus/0.37.0", map[string]string{
-		"olm.bundle.path": "quay.io/operatorhubio/prometheus@sha256:3e281e587de3d03011440685fc4fb782672beab044c1ebadc42788ce05a21c35",
+		"olm.bundle.path": `"quay.io/operatorhubio/prometheus@sha256:3e281e587de3d03011440685fc4fb782672beab044c1ebadc42788ce05a21c35"`,
```

Comment:
This file still has some double quoting going on, unclear if expected?

Reply:
@perdasilva so do I need to put get rid of the quotes (that the revert adds in)? Fwiw the unit test cases pass without any problem now.

Reply:
Leave as is. I think it's expected to be a JSON string, which needs to retain quotes.